src/plat/odroidc4/config.cmake

```
@@ -17,7 +17,8 @@ if(KernelPlatformOdroidc4)
     list(APPEND KernelDTSList "tools/dts/odroidc4.dts" "src/plat/odroidc4/overlay-odroidc4.dts")
     declare_default_headers(
         TIMER_FREQUENCY 24000000
-        MAX_IRQ 250
+        # MAX_IRQ is based on the section 7.10.2 of the S905X3 SoC manual
+        MAX_IRQ 255
```

For what it's worth, this will pass the style check.
```suggestion
+        MAX_IRQ
+            255 # based on section 7.10.2 of the S905X3 SoC manual
```

---

That looks pretty horrible too.